Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -183,13 +183,14 @@ clusters.
 
 .. note::
 
-   You can additionally opt in to :ref:`kvstoremesh` when enabling
-   Cluster Mesh. Make sure to configure the Cilium CLI in *helm* mode and run:
+   Starting from v1.16 KVStoreMesh is enabled by default.
+   You can opt out of :ref:`kvstoremesh` when enabling
+   Cluster Mesh. Make sure to configure the Cilium CLI in *helm* mode and run instead:
 
    .. code-block:: shell-session
 
-     cilium clustermesh enable --context $CLUSTER1 --enable-kvstoremesh
-     cilium clustermesh enable --context $CLUSTER2 --enable-kvstoremesh
+     cilium clustermesh enable --context $CLUSTER1 --enable-kvstoremesh=false
```

Comment:
Just a note that this will require a symmetric change in the Cilium CLI to not set the `kvstoremesh.enabled` helm value if the user didn't explicitly use `--enable-kvstoremesh[=false]`. The reason being that we cannot just flip the default there, as dependent on the target Cilium's version.

Reply:
yea, that's why I prefer to explicitly set it here and follow up with cilium-cli change to make it use default if it's not set.